tests/integration/test_vulnerability_detector/test_general_settings/test_min_full_scan_interval.py

```diff
@@ -12,10 +12,16 @@
        and allows to define the minimum time before performing a full scan even if the feed was updated. The extended
        behavior also checks that the full scan is performed as expected.
 
+<<<<<<< HEAD
```

Comment:
Conflicts have not been properly resolved

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/2649/commits/71e4587f39df3f93b1e16f8387b2c14fb5bc217b